src/common/uuid.dto.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { validate } from 'class-validator';
+import { UuidDto } from './uuid.dto';
+
+describe('UuidDtoDto Validation', () => {
+  it('should pass validation with a valid UUID', async () => {
```

Comment:
Remember the "When xxxx, then xxx" structure. I know this is just for a dto, but just to be consistent 👍🏻

Reply:
When a valid UUID is passed, then pass.

When an invalid UUID is passed, then fail.

When an empty string is passed, then fail.